src/dynatrace/opentelemetry/metrics/export/_exporter.py

```diff
@@ -30,8 +31,12 @@
 )
 from opentelemetry.sdk.metrics.view import Aggregation
 
+from dynatrace.opentelemetry.metrics.export._constants import (
+    _DYNATRACE_TEMPORALITY_PREFERENCE,
+)
+
 from dynatrace.opentelemetry.metrics.export._factory import (
-    _OTelDynatraceMetricsFactory
+    _OTelDynatraceMetricsFactory,
```

Comment:
nit: should we apply this style (trailing `,`) to all imports in this file? :thinking: